src/request/RequestClient.ts

```diff
@@ -419,7 +420,13 @@ export class RequestClient implements HttpClient {
       }
     } catch {
       try {
-        parsedResponse = JSON.parse(parseWeboutResponse(response.data))
+        const weboutResponse = parseWeboutResponse(response.data)
+        if (weboutResponse === '') {
+          throw new Error('Valid JSON could not be extracted from response.')
+        }
+        console.log('RequestClient')
```

Comment:
I think we can remove this (and other) console.log statements that might be left over.

Reply:
removed